repository-data/application/src/main/resources/hcm-content/content/documents/administration/facet-labels.yaml

```diff
@@ -81,7 +81,7 @@
     - Publication
     - Data set
     - Series / Collection
-    - Indicator
+    - Methodology
```

Comment:
Note you'll have to change this manually on UAT / Prod as yaml files only get loaded once